docs/man

```diff
@@ -0,0 +1,41 @@
+# The man Command
+
+## Summary 
+The `man` command in Linux is used to display the user manual of any command that we can run on
+the terminal. It provides a detailed view of the command which includes NAME, SYNOPSIS,
+DESCRIPTION, OPTIONS, EXIT STATUS, RETURN VALUES, ERRORS, FILES, VERSIONS, EXAMPLES,
+AUTHORS and SEE ALSO.
+https://www.nielit.gov.in/gorakhpur/sites/default/files/Gorakhpur/Alevel_unix_27april20_AKM.pdf
```

Comment:
URLs should be correctly formatted as hyperlinks using the Markdown syntax described in the documentation here: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links

Reply:
@les542 This hasn't been resolved yet